### PR TITLE
Add EU Digital Assembly blog post

### PR DESCRIPTION
**Pull request by ch-ahindura**
### Why

Chalmers University of Technology had a live demo in the EU Digital Assembly in June 2023 that showcased some of the work done in NordIQuEst. This is an attempt to document it.

### What was Done

- Added images for the blog post
- Added the blog post page

### CODE REVIEW

File: _posts/2024-05-02-EU-Digital-Assembly-2023.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+Dr. Dobsicek's live demonstration showcased two crucial stages involved in this innovative application.  
+ 
+At the outset, he demonstrated a live calibration measurement of the QAL 9000 processor. Quantum computing applications rely on incredibly precise control of quantum states.  
+
+To achieve this level of precision, Dr. Dobsicek utilized QAL 9000’s cloud RESTful API interface to perform a live recalibration measurement called a π-pulse. This specific pulse drives the transition of qubits, the quantum bits that hold information, between the quantum states of 0 and 1. With this recalibration a noticeable better state discrimination has been achieved.
+
+{% include figure image_path="/assets/images/eu-digital-assembly-qal9000-live-tuneup.png" alt="A Screenshot of the live tune up of QAL 9000." caption="A Screenshot of the live tune up of QAL 9000. Credit: Martin Ahindura" %}
+
+Of note is that the calibration was performed concurrently on a significant portion of the processor, focusing on12 out of the 25 total qubits available on the QAL 9000 chip.
```
Comment (JMuff22) [inline _posts/2024-05-02-EU-Digital-Assembly-2023.md]:
```suggestion
Of note is that the calibration was performed concurrently on a significant portion of the processor, focusing on 12 out of the 25 total qubits available on the QAL 9000 chip. 
```

File: _posts/2024-05-02-EU-Digital-Assembly-2023.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+ 
+At the outset, he demonstrated a live calibration measurement of the QAL 9000 processor. Quantum computing applications rely on incredibly precise control of quantum states.  
+
+To achieve this level of precision, Dr. Dobsicek utilized QAL 9000’s cloud RESTful API interface to perform a live recalibration measurement called a π-pulse. This specific pulse drives the transition of qubits, the quantum bits that hold information, between the quantum states of 0 and 1. With this recalibration a noticeable better state discrimination has been achieved.
+
+{% include figure image_path="/assets/images/eu-digital-assembly-qal9000-live-tuneup.png" alt="A Screenshot of the live tune up of QAL 9000." caption="A Screenshot of the live tune up of QAL 9000. Credit: Martin Ahindura" %}
+
+Of note is that the calibration was performed concurrently on a significant portion of the processor, focusing on12 out of the 25 total qubits available on the QAL 9000 chip. 
+
+The state discrimination has improved after the recalibration; Less blue dots, |1> state, we present in the red point-could.
```
Comment (JMuff22) [inline _posts/2024-05-02-EU-Digital-Assembly-2023.md]:
```suggestion
The state discrimination has improved after the recalibration; fewer blue dots, |1> state, we present in the red point-cloud. 
```

Comment (JMuff22) [reply _posts/2024-05-02-EU-Digital-Assembly-2023.md]: I am not sure why there is a line in the page at this point.

**Review by JMuff22 — COMMENTED** [review]
Looks good Martin! Some typos here and there to change. There seems to be a page break which is causing formatting errors. Let me see if I can fix.

File: _posts/2024-05-02-EU-Digital-Assembly-2023.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+{% include figure image_path="/assets/images/eu-digital-assembly-qal9000-live-tuneup.png" alt="A Screenshot of the live tune up of QAL 9000." caption="A Screenshot of the live tune up of QAL 9000. Credit: Martin Ahindura" %}
+
+Of note is that the calibration was performed concurrently on a significant portion of the processor, focusing on12 out of the 25 total qubits available on the QAL 9000 chip. 
+
+The state discrimination has improved after the recalibration; Less blue dots, |1> state, we present in the red point-could. 
+
+After the successful recalibration, Dr. Dobsicek demonstrated a basic machine 
+learning algorithm tailored to classify points within a two-dimensional plane. The algorithm was based on the concepts described in [1]. 
+
+{% include figure image_path="/assets/images/eu-digital-assembly-quantum-neural-network.png" alt="Diagram showing A quantum neural network in QAL 9000 with a classical optimizer in LUMI." caption="A quantum neural network in QAL 9000 with a classical optimizer in LUMI. Credit: Dr. Miroslav dobsicek" %}
```
Comment (JMuff22) [inline _posts/2024-05-02-EU-Digital-Assembly-2023.md]:
```suggestion
{% include figure image_path="/assets/images/eu-digital-assembly-quantum-neural-network.png" alt="Diagram showing A quantum neural network in QAL 9000 with a classical optimizer in LUMI." caption="A quantum neural network in QAL 9000 with a classical optimizer in LUMI. Credit: Dr. Miroslav Dobsicek" %}
```

File: _posts/2024-05-02-EU-Digital-Assembly-2023.md (hunk 2 above)
Comment (JMuff22) [reply _posts/2024-05-02-EU-Digital-Assembly-2023.md]: It seems that in this case jekyll does not like `|1>` even with Kramdown+Mathjax. Could you say "the first excited state" or the excited state instead?

Comment (Tinitto) [reply _posts/2024-05-02-EU-Digital-Assembly-2023.md:44]: I could just do `|1>`

**Review by ch-ahindura — COMMENTED** [review]
Fixed

**Review by JMuff22 — APPROVED** [review]
LGMT!

**Pull request merged**